null_safety/README.md

```diff
@@ -0,0 +1,97 @@
+# Null safety Flutter example
+
+This folder contains a demonstration of the upcoming Dart feature, sound null
+safety. For additional information about this feature, please see
+https://dart.dev/null-safety.
+
+The folder contains two sample Flutter apps:
+
+  * `null_unsafe_app`: This contains a small weather app, which has three null
+    error in it's rendering code, as a result of not using null safety.
+    
+    Try running the app (`flutter run in a terminal, or the Run action in
+    VSCode/Android Studio), and see if the UI works. The null errors are
+    spurious, so try hot reloading a bunch of times and experience the app
```

Comment:
Can we use a different word than "spurious". I had to look up that it can also mean:
> of a deceitful nature or quality

to understand what was meant. Maybe "sporadic"? Maybe something better.